phpstan-baseline.neon

```diff
@@ -0,0 +1,3161 @@
+parameters:
```

Comment:
This generated PHP code is so bad, I am simply not going to bother trying to fix it

Reply:
😢